core/controllers/acl_decorators_test.py

```diff
@@ -1859,6 +1859,7 @@ def setUp(self):
             [webapp2.Route('/mock_edit_topic/<topic_id>', self.MockHandler)],
             debug=feconf.DEBUG,
         ))
+        self.topic_id = topic_services.get_new_topic_id()
```

Comment:
@kevintab95 I just wanted to check how it was working earlier?

Reply:
We didn't have the topic name length check in the acl decorator before, so it passed.